### PR TITLE
Add Dazhong Dianping icon

### PR DESCRIPTION
**Pull request by LitoMore**
CleanShot 2023-12-16 at 15 20 26@2x

**Similarweb rank:** [6393](https://www.similarweb.com/website/dianping.com/#overview)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

I picked the resource from their media kit.

It is Chinese largest life style platform where users directly upload their pictures, videos and reviews, and share each other's experience in the world.

### CODE REVIEW

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
I'm seeing some differences throughout this one, @LitoMore. Most pronounced being [this one](https://github.com/simple-icons/simple-icons/assets/15157491/d9907b94-381e-4d98-983a-2e30c4315983) in the circle. Likely just a precision issue.

File: icons/dazhongdianping.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Dazhong Dianping</title><path d="M6.924 0C4.517 0 3.644.25 2.76.718A4.906 4.906 0 0 0 .72 2.763C.248 3.643 0 4.517 0 6.92v10.156c0 2.407.25 3.28.718 4.161a4.919 4.919 0 0 0 2.041 2.041c.884.473 1.758.722 4.165.722h10.152c2.407 0 3.28-.25 4.161-.718a4.906 4.906 0 0 0 2.041-2.045c.473-.88.722-1.754.722-4.157V6.92c0-2.407-.25-3.28-.718-4.16A4.906 4.906 0 0 0 21.24.717C20.353.25 19.48 0 17.072 0Zm3.092 2.706a3.07 3.07 0 1 1 .094 6.138 3.07 3.07 0 0 1-.094-6.138zm7.037 1.614a6.64 6.64 0 0 1 .133 2.056 6.826 6.826 0 0 1-2.48 4.656 7.824 7.824 0 0 1 6.512 5.5h.008l.212.725-2.688.684-.189-.669c-.756-2.623-3.552-4.172-6.274-3.47-2.721.708-4.339 3.395-3.643 6.025l.196.73-2.683.684-.204-.752h.007a7.381 7.381 0 0 1-.215-2.555 7.555 7.555 0 0 1 3.012-5.36A7.083 7.083 0 0 1 3.25 7.9l2.62-.68c.567 1.538 2.014 2.69 3.802 2.842 2.445.208 4.603-1.542 4.819-3.908a4.15 4.15 0 0 0-.057-1.149Z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/dazhongdianping.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Dazhong Dianping</title><path d="M6.9226 0h10.1549c2.407 0 3.2801.2506 4.1602.7214.8798.4705 1.5704 1.1611 2.041 2.0412.4707.88.7213 1.7529.7213 4.16v10.1549c0 2.4073-.2506 3.2802-.7214 4.16-.4705.88-1.1611 1.5707-2.041 2.0414-.88.4705-1.753.7212-4.1601.7212H6.9225c-2.407 0-3.2799-.2507-4.16-.7212-.88-.4707-1.5706-1.1614-2.0414-2.0414C.2506 20.3577 0 19.4848 0 17.0775V6.9225c0-2.407.2506-3.2799.7211-4.16.4708-.88 1.1614-1.5706 2.0415-2.0411C3.6426.2506 4.5155 0 6.9226 0m.0806 5.5064c.1475-1.6889 1.6364-2.9383 3.3256-2.7903 1.689.1475 2.938 1.6364 2.7903 3.3256-.148 1.6886-1.6366 2.9383-3.3255 2.7903-1.6892-.1472-2.9383-1.6364-2.7904-3.3256Zm14.4332 11.7497-2.6875.6857c-.0608-.2227-.129-.4519-.191-.6696-.7518-2.6225-3.5493-4.1736-6.2722-3.4676-2.7184.7045-4.3385 3.3922-3.6431 6.0222l.1988.7288-2.6875.6862c-.0693-.2476-.1328-.5125-.1953-.7538a7.3818 7.3818 0 0 1-.2137-2.5551c.1988-2.1996 1.3431-4.099 3.012-5.3581-2.634-.4693-4.715-2.3124-5.5046-4.6781l2.6193-.6788c.5667 1.5343 2.013 2.6892 3.801 2.8402 2.4463.207 4.6035-1.5432 4.8174-3.9096a4.1722 4.1722 0 0 0-.056-1.1498l2.6208-.679a6.7095 6.7095 0 0 1 .1309 2.0574c-.1703 1.88-1.1044 3.5191-2.4798 4.6555 3.1769.4264 5.6846 2.6493 6.5133 5.5004.0807.2201.1485.486.2172.7231z"/></svg>
```

File: icons/dazhongdianping.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...c2.407 0 3.2801.2506 4.1602.7214.8798.4705 1.5704 1.1611 2.041 2.0412.4707.88.7213 1.7529.7213 4.16v10.1549c0 2.4073-.2506 3.2802-.7214 4.16-.4705.88-1.1611 1.5707-2.041 2.0414-.88.4705-1.753.7212-4.1601.7212H6.9225c-2.407 0-3.2799-.2507-4.16-.7212-.88-.4707-1.5706-1.1614-2.0414-2.0414C.2506 20.3577 0 19.4848 0 17.0775V6.9225c0-2.407.2506-3.2799.7211-4.16.4708-.88 1.1614-1.5706 2.0415-2.0411C3.6426.2506 4.5155 0 6.9226 0m.0806 5.5064c.1475-1.6889 1.6364-2.9383 3.3256-2.7903 1.689.1475 2.938 1.6364 2.7903 3.3256-.148 1.6886-1.6366 2.9383-3.3255 2.7903-1.6892-.1472-2.9383-1.6364-2.7904-3.3256Zm14.4332 11.7497-2.6875.6857c-.0608-.2227-.129-.4519-.191-.6696-.7518-2.6225-3.5493-4.1736-6.2722-3.4676-2.7184.7045-4.3385 3.3922-3.6431 6.0222l.1988.7288-2.6875.6862c-.0693-.2476-.1328-.5125-.1953-.7538a7.3818 7.3818 0 0 1-.2137-2.5551c.1988-2.1996 1.3431-4.099 3.012-5.3581-2.634-.4693-4.715-2.3124-5.5046-4.6781l2.6193-.6788c.5667 1.5343 2.013 2.6892 3.801 2.8402 2.4463.207 4.6035-1.5432 4.8174-3.9096a4.1722 4.1722 0 0 0-.056-1.1498l2.6208-.679a6.7095 6.7095 0 0 1 .1309 2.0574c-.1703 1.88-1.1044 3.5191-2.4798 4.6555 3.1769.4264 5.6846 2.6493 6.5133 5.5004.0807.2201.1485.486.2172.7231z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/dazhongdianping.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Dazhong Dianping</title><path d="M6.922 0c-2.407 0-3.28.25-4.16.72A4.905 4.905 0 0 0 .72 2.763C.25 3.642 0 4.515 0 6.922v10.156c0 2.407.25 3.28.72 4.16.471.88 1.162 1.57 2.042 2.041.88.47 1.753.721 4.16.721h10.156c2.407 0 3.28-.25 4.16-.72a4.907 4.907 0 0 0 2.041-2.042c.471-.88.721-1.753.721-4.16V6.922c0-2.407-.25-3.28-.72-4.16A4.905 4.905 0 0 0 21.237.72C20.358.25 19.485 0 17.078 0H6.922zm3.094 2.705a3.075 3.075 0 0 1 3.103 3.336 3.068 3.068 0 1 1-3.104-3.336zm7.039 1.615c.145.66.195 1.35.13 2.057-.17 1.88-1.105 3.52-2.48 4.656 3.177.427 5.685 2.65 6.514 5.5.08.22.148.486.217.723l-2.688.685c-.06-.222-.127-.452-.19-.67-.751-2.622-3.55-4.172-6.273-3.466-2.718.704-4.338 3.391-3.642 6.021l.199.729-2.688.687c-.07-.247-.133-.512-.195-.754a7.38 7.38 0 0 1-.215-2.554c.199-2.2 1.345-4.1 3.014-5.36-2.634-.47-4.716-2.312-5.506-4.678l2.62-.677c.566 1.534 2.012 2.689 3.8 2.84 2.446.207 4.604-1.544 4.818-3.91.036-.394.013-.78-.056-1.151l2.62-.678z"/></svg>
```

**Review by service-paradis — APPROVED** [review]
LGTM 🎉

**Pull request merged**